### PR TITLE
Add icon

### PR DESCRIPTION
**Pull request by hafizahmmed**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+![asset 1-8](https://user-images.githubusercontent.com/38810169/40872729-06168d0e-6675-11e8-9537-af0a0f133d11.png)
```
Comment (xRahul) [inline README.md]: You can add `NotifyAvailability` instead of `asset 1-8` and remove the header below. This by itself looks good enough

Comment (hafizahmmed) [reply README.md]: i removed this

**Comment by hafizahmmed** [comment]
now check

File: README.md (hunk 1 above)
Comment (xRahul) [reply README.md]: This was actually a good header image. Can you add this image in the path `images/readmeHeader.png`?

File: images/icon [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (xRahul) [inline images/icon]: This file is unnecessary. I was asking to move the entire icon folder that you have created in the folder images.

**Comment by xRahul** [comment]
Also, can you add this [image](https://user-images.githubusercontent.com/38810169/40872729-06168d0e-6675-11e8-9537-af0a0f133d11.png) in the path `images/readmeHeader.png`?
I could use this as the header image for this app.

**Pull request merged**